strings/base_coroutine_threadpool.h

```diff
@@ -396,17 +409,10 @@ namespace winrt::impl
         }
 
     private:
-        void create_threadpool_timer()
+        void set_threadpool_timer()
         {
-            m_timer.attach(check_pointer(WINRT_IMPL_CreateThreadpoolTimer(callback, this, nullptr)));
             int64_t relative_count = -m_duration.count();
             WINRT_IMPL_SetThreadpoolTimer(m_timer.get(), &relative_count, 0, 0);
-
-            state expected = state::idle;
-            if (!m_state.compare_exchange_strong(expected, state::pending, std::memory_order_release))
-            {
-                fire_immediately();
-            }
         }
 
         static int32_t __stdcall fallback_SetThreadpoolTimerEx(winrt::impl::ptp_timer, void*, uint32_t, uint32_t) noexcept
```

Comment:
Probably want to consider dropping Windows 7 support some time as well. That could further simplify this stuff a bit.

Reply:
We already dropped support for Win7 RTM in #1293. We now require KB2533623 at a minimum. (Mind you, KB2533623 was released in 2011, so anybody running Win7 will have it if they have been paying even the slightest bit of attention.)

Reply:
Yep, I meant the numerous calls to `load_runtime_function` that still exist in `base.h` should probably just get yanked out for simplicity.

Reply:
But that's not specific to this PR so I'll just reference this in a new issue.